shuttle/templates/nav.html

```diff
@@ -1,7 +1,8 @@
 <nav class="navbar navbar-expand-lg navbar-dark background-blue">
     <a class="navbar-brand" href="{{ url_for('View:index') }}">
         <img height="48" src="https://confluence.bethel.edu/download/attachments/327682/global.logo?version=4&amp;modificationDate=1462424762000&amp;api=v2">
-        <span class="ml-2  h6 lab-title">Shuttle</span>
+        <a href="/"><span class="ml-2  h6 lab-title" style="color:white">Shuttle</span></a>
+        <a href="/shuttle-stats"><span class="ml-2  h6 lab-title background-blue" style="color:white">Shuttle Stats</span></a>
```

Comment:
Change the href to use url_for just like on line 2 of this file.

Reply:
Ahh good idea. I changed those